packages/edit-site/src/components/dataviews/filters.js

```diff
@@ -34,27 +36,37 @@ export default function Filters( { fields, view, onChangeView } ) {
 		} );
 	} );
 
-	return view.visibleFilters?.map( ( fieldName ) => {
-		const visibleFiltersForField = filtersRegistered.filter(
-			( f ) => f.field === fieldName
-		);
+	const visibleFilters = view.visibleFilters
+		?.map( ( fieldName ) => {
+			const visibleFiltersForField = filtersRegistered.filter(
+				( f ) => f.field === fieldName
+			);
 
-		if ( visibleFiltersForField.length === 0 ) {
-			return null;
-		}
-
-		return visibleFiltersForField.map( ( filter ) => {
-			if ( OPERATOR_IN === filter.operator ) {
-				return (
-					<InFilter
-						key={ fieldName }
-						filter={ visibleFiltersForField[ 0 ] }
-						view={ view }
-						onChangeView={ onChangeView }
-					/>
-				);
+			if ( visibleFiltersForField.length === 0 ) {
+				return null;
 			}
-			return null;
-		} );
-	} );
+
+			return visibleFiltersForField.map( ( filter ) => {
+				if ( OPERATOR_IN === filter.operator ) {
+					return (
+						<InFilter
+							key={ fieldName }
+							filter={ visibleFiltersForField[ 0 ] }
+							view={ view }
+							onChangeView={ onChangeView }
+						/>
+					);
+				}
+				return null;
+			} );
+		} )
+		.filter( Boolean );
+
+	if ( visibleFilters.length > 0 ) {
+		visibleFilters.push(
+			<ResetFilters view={ view } onChangeView={ onChangeView } />
```

Comment:
Since we're returning an array of items, we should have a `key` prop here.

Reply:
Fixed at https://github.com/WordPress/gutenberg/pull/55955/commits/a5ffee5a518a301719f65a19614c0883e4d0aab7